s2wav/Dimension helper functions.py

```diff
@@ -0,0 +1,370 @@
+# This file translates functions which can be called to check dimensions of vectors from s2let (c).
+
+from cmath import log
+from functools import total_ordering
+import math
+from tkinter import N
+
+
+#Defining functions originally called from ssht
+
+
+def ssht_sampling_mw_ss_nphi(L: int) -> int:
+    '''Computes the number of phi samples for McEwen and Wiaux symmetric sampling.
+    
+    Args:
+        L: Upper harmonic band-limit.
+
+    Returns:
+        nphi: Number of phi samples.   
+    '''
+    return 2*L
+
+
+
+def ssht_sampling_mw_nphi(L: int) -> int:
+    '''Computes the number of phi samples for McEwen and Wiaux sampling.
+    
+    Args:
+        L: Upper harmonic band-limit.
+
+    Returns:
+        nphi: Number of phi samples.
+    '''
+    return 2*L-1
+
+
+
+def ssht_sampling_mw_ss_ntheta(L: int) -> int:
+    '''Computes the number of theta samples for McEwen and Wiaux symmetric sampling.
+    
+    Note: Computes the number of samples in [0, pi], *not* over extended domain.
+
+    Args:
+        L: Harmonic band-limit.
+    
+    Returns:
+        ntheta: Number of theta samples.   
+    '''
+    return L+1
+
+
+
+def ssht_sampling_mw_ntheta(L: int) -> int:
+    '''Computes the number of theta samples for McEwen and Wiaux sampling.
+
+    Note: Computes the number of samples in (0, pi], *not* over extended domain.
+
+    Args:
+        L: Harmonic band-limit.
+    
+    Returns:
+        ntheta: Number of theta samples.
+    '''
+    return L
+
+
+
+
+#Functions from lines 41 - 110 of s2let_tiling.c
+
+
+
+def s2let_bandlimit(s2let_kernel: str, lam: float, j: int) -> int:
+    '''Computes the band-limit of a specific wavelet scale.
+
+    Args:
+        s2let_kernel: The wavelet type.
+        lam: Wavelet parameter which determines the scale factor between consecutive wavelet scales.
+        j: Wavelet scale.
+    
+    Returns:
+        Band-limit.
```

Comment:
Returns should be formatted as


`data_type: description of what is returned`

So something like

`int: spherical harmonic band-limit`

We should make sure this is consistent throughout

Reply:
Like this?
`returns:
         data_type: description`